nexus/operation.go

```diff
@@ -15,6 +15,31 @@ import (
 //	)}
 type NoValue *struct{}
 
+// OperationOptions contains the general options for an operation, across different handlers.
+//
+// NOTE: Experimental
+type OperationOptions struct {
+	// ServiceName is the name of the service that contains the operation.
+	ServiceName string
+	// OperationName is the name of the operation.
+	OperationName string
```

Comment:
I think we can drop the `Name` suffix here but don't have a strong opinion.

Reply:
Done